src/extension.ts

```diff
@@ -24,6 +24,11 @@ export function activate(context: vscode.ExtensionContext) {
         const selection = editor.selection;
         const text = document.getText(selection);
 
+        if (text.trim() === "") {
+          vscode.window.showErrorMessage("YOU HAVE NOT HIGHLIGHTED ANYTHING.");
```

Comment:
Error message shouldn't be in caps.

Reply:
Done. It's now "You have not highlighted anything."